Formatting updates using newest version of clang-format

oddbookworm opened the pull request.

ankith26 commented on src_c/freetype/ft_wrap.c:
```diff
@@ -38,8 +38,7 @@ ft_wrap_quit(pgFontObject *);
 void
-_PGFT_SetError(FreeTypeInstance *ft, const char *error_msg, FT_Error error_id)
-{
+_PGFT_SetError(FreeTypeInstance *ft, const char *error_msg, FT_Error error_id){
```
The changes in this file are weird... but we will just go with whatever clang-format is doing here

ankith26 reviewed (approved):
LGTM, thanks for the PR

ankith26 commented:
There seems to be a new mypy fail too, as it catches a real issue. Maybe that can be handled in this PR as well, just gotta add something like
```py
def colorspace(
    surf: Surface, color: Literal["YUV", "HSV"], dest_surf: Surface = ..., /
) -> Surface: ...
```
to `camera.pyi`

oddbookworm commented:
> There seems to be a new mypy fail too, as it catches a real issue. Maybe that can be handled in this PR as well, just gotta add something like
> 
> ```python
> def colorspace(
>     surf: Surface, color: Literal["YUV", "HSV"], dest_surf: Surface = ..., /
> ) -> Surface: ...
> ```
> 
> to `camera.pyi`

Updated. I also updated the doc signature to match
